WcaOnRails/app/views/posts/_specialbox.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="ui card">
   <div class="content">
-    <h3 class="center aligned header"><%= title %></h3>
+    <h3 class="center aligned header"><%= (title+"<span class=\"blink\"><sup style=\"color: #ff0000;font-size: 10px;\">NEW</sup></span>").html_safe%></h3>
```

Comment:
Why are you mangling the HTML tags into a Ruby string, instead of adding them in the HTML view directly?
`<h3><%= title + '<foo></foo>' ></h3>` should become `<h3><%= title ><foo></foo></h3>`

Reply:
You're basically wrapping HTML tags as a Ruby string only to immediately unwrap them again

Reply:
Also, while I'm at it: Don't do inline `style: xyz` directly in the HTML. Give a `class` like `red-footnote` to the `sup` and then in CSS do
```
.red-footnote {
   xyz
}
```